src/Style.js

```diff
@@ -6,6 +6,18 @@
  * information about them.
  */
 
+var sigmas = require("./fontMetrics.js").sigmas;
+
+var metrics = [];
+
+for (var i = 0; i < 3; i++) {
+    metrics[i] = Object.keys(sigmas).reduce(function(res, key) {
```

Comment:
According to MDN, [`Array.prototype.reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) and [`Object.keys`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) require IE 9, while KaTeX officially supports IE 8. So this should probably be reformulated using something like

```js
var metrics = [{}, {}, {}];
for (var key in sigmas)
    for (var i = 0; i < 3; i++)
        metrics[i][key] = sigmas[key][i];
for (var i = 0; i < 3; i++)
    metrics[i].emPerEx = sigmas.xHeight[i] / sigmas.quad[i];
```

I wonder whether we should add some tool to our build pipeline to either catch such issues or polyfill required functionality. Or do we already have a polyfill, and I wasn't aware of it? `package.json` doesn't list any obvious candidates.